enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,335 @@
+      status: 'True'
+      type: PlacementSatisfied
+```  
+The created placementDecisions will have ClusterGroupIndex label to relate with the cluster groups as below.
+
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: ztp-placement-decision-0
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: ztp-placement
+    cluster.open-cluster-management.io/clustergroupindex: 0
+status:
+  decisions:
+    - clusterName: cls1
+      reason: ''
+    ...
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: ztp-placement-decision-1
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: ztp-placement
+    cluster.open-cluster-management.io/clustergroupindex: 1
+status:
+  decisions:
+    - clusterName: cls001
+      reason: ''
+    ...
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: PlacementDecision
+metadata:
+  name: ztp-placement-decision-3
+  namespace: ztp-acm-ns
+  labels:
+    cluster.open-cluster-management.io/placement: ztp-placement
+    cluster.open-cluster-management.io/clustergroupindex: 2
+status:
+  decisions:
+    - clusterName: cls151
+      reason: ''
+    ...
+```
+
+The Placement Helper [library](https://github.com/open-cluster-management-io/api/blob/main/cluster/v1beta1/helpers.go) should provide functions to facilitate retrieve the cluster groups.
+
+#### Special case handling
+
+* A cluster has been removed from group index 1 and group index 2 has been created. In order to not shift current selected clusters groups, placement controller will keep append clusters to group index 2 till it reach max number of clusters per group then append added clusters to group index 1 till it reach the max number of clusters per group.
+
+
+### How Workload applier APIs will benefits from using placementStrategy
+
+The OCM APIs such as ManifestWorkReplicaSet will check the PlacementStatus APIs to determine how many cluster groups exist in order to rollout the manifestWork. Let's consider the below ManifestWorkReplicaSet example has a placementRef (placement reference) to the above ztp-placement example.
+
+```yaml
+apiVersion: work.open-cluster-management.io/v1alpha1
+kind: ManifestWorkReplicaSet
+metadata:
+  name: ztp-mwrset
+  namespace: ztp-acm-ns
+spec:
+  placementRefs:
+    - name: ztp-placement
+  manifestWorkTemplate:
+    ...
+```
+The ManifestWorkReplicaSet controller will call the placement API to retrieve the cluster groups. In the ztp-placement example there are 3 cluster groups.
+```yaml
+  clusterGroups:
+  - clusterGroupIndex: 0
+    clusterGroupType: MandatoryClusterGroup
+    placementDecisions:
+    - ztp-placement-decision-0
+    clusterCount: 10
+  - clusterGroupIndex: 1
+    clusterGroupType: ProgressiveClusterGroup
+    placementDecisions:
+    - ztp-placement-decision-1
+    - ztp-placement-decision-2
```

Comment:
when the clusters in each placementDecision are changed, for example some clusters removed from the placement, some new clusters are added into the placement, the clusters in each group will be re-grouped or not ?

Reply:
It should not re-grouped when a new cluster added it will be appended to last group (or create new group if last one reach its max capacity).  Check the special case handling section.